_positions/product-manager.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Product Manager
+permalink: positions/product-manager/
+team: 18F
+active: true
+---
+##Basic Information:
+**Title:** Product Manager  <br>
```

Comment:
For future reference, Markdown files (files that have a `.md` extension, like this one) don't use `<br>`. All they need for a linebreak is a linebreak. I'll go ahead and take these out.